packages/slider/src/spectrum-slider.css

```diff
@@ -872,72 +727,60 @@ governing permissions and limitations under the License.
             var(--spectrum-slider-tick-mark-height)
     );
 }
-.tick:first-of-type .tickLabel,
-.tick:last-of-type .tickLabel {
-    display: block; /* .spectrum-Slider-tick:first-of-type .spectrum-Slider-tickLabel,
-   * .spectrum-Slider-tick:last-of-type .spectrum-Slider-tickLabel */
+.tick:first-of-type .tick-label,
```

Comment:
Looks like you used `tick-label` instead of `tickLabel`. I prefer the kabob, generally, but it'd likely require JS changes, too.